src/paymentservice/charge.js

```diff
@@ -15,6 +15,8 @@
 const cardValidator = require('simple-card-validator');
 const { v4: uuidv4 } = require('uuid');
 const pino = require('pino');
+const opentelemetry = require('@opentelemetry/api');
+const tracer = opentelemetry.trace.getTracer("");
```

Comment:
Is it a good practice to leave the trace unnamed?

Reply:
I'm actually not certain about this. I know you can have more than 1 tracer, though I'm not clear on the use cases to have more than 1 tracer. 

I believe if you pass in an empty name you will get the default tracer. If I'm always going to get the default tracer with a blank string, I would prefer to promote that as a practice, rather than a name, which could be prone to a typo.

Reply:
> if you pass in an empty name you will get the default tracer

If that's true, I'm ALL IN!

I have never seen that info in any documentation though. Who could confirm that for us?

Reply:
I did some more digging into this, and I found that we should probably specify a name. Though it is true a default tracer will be returned by all SDKs, some will also log an error if you pass an empty name, and I don't think that's an experience we want people to have.

I think we should change this to use the service name for each one instead. I may need to go back and update some of the services we already did, so this may be better served as a separate Issue and PRs to track against.

Reply:
Awesome @puckpuck!